src/email/mailchimp/audience_analysis.R

```diff
@@ -0,0 +1,563 @@
+box::use(
+  src/email/mailchimp/audience,
+  src/email/mailchimp/base_api
+)
+
+box::use(
+  dplyr,
+  gg = ggplot2,
+  gghdx,
+  httr2,
+  lubridate,
+  purrr,
+  rnaturalearth,
```

Comment:
`{rnaturalearth}` is not a signals dependency (not in renv lockfile), may not be worth adding if it it won't be used in any pipeline